Automattic/pocket-casts-ios
# Tracks: Add Firebase opt out

| 📘 Project: #154 |
|:---:|

Adds check for Firebase opt out. 

## To test

1. Enable the `firebaseLoggingEnabled` FeatureFlag
2. Launch the app
3. Tap around the app
4. Verify you see `🟢 Tracked:` logs
5. Go to Profile > Cog > Privacy
6. Enable the opt out
7. Tap around
8. Verify you don't see `🟢 Tracked:` logs

## Checklist
